fuzzers/nyx_libxml2_standalone/README.md

```diff
@@ -0,0 +1,16 @@
+this example shows to use `libafl_nyx` to fuzz `libxml2`
+
+# requirement
```

Comment:
Also would be good to have a `Makefile.toml` here

Reply:
fixed